Make sure the perl code is valid both with and without the backslashes.
Some make versions strip them, others don't.

--- glastree-1.04/Makefile
+++ glastree-1.04/Makefile
@@ -24,8 +24,8 @@
 
 install:
 	@perl -e 'eval "use Date::Calc"; \
-	print "Checking for prereq Date::Calc..."; \
-	if ($$@) { print "\nThis program requires the Date::Calc module.\n"; exit 1; } \
+	print "Checking for prereq Date::Calc..."; if ($$@) { \
+	print "\nThis program requires the Date::Calc module.\n"; exit 1; } \
 	print " OK\n"; '
 
 	$(INSTALL) -d $(INSTROOT)/$(INSTBIN)
